theme6/source/javascripts/featured_products.js

```diff
@@ -0,0 +1,26 @@
+var images = ["bear_toy", "train_toy", "plane_toy", "ship_toy"];
+var i = 0;
+
+var leftArrow = document.getElementById("leftArrow");
+var rightArrow = document.getElementById("rightArrow");
+
+this.leftArrow.addEventListener('click', () => {
+  changeImage(-1);
+});
+
+this.rightArrow.addEventListener('click', () => {
```

Comment:
Mixed double and single quotes.<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).